addons/purchase_stock/tests/test_purchase_order_suggest.py

```diff
@@ -51,15 +51,13 @@ def actionAddAll(self, po, based_on='30_days', days=30, factor=100, warehouse=Fa
         base_warehouse = self.picking_type_out.default_location_src_id.warehouse_id
         warehouse_id = (warehouse or base_warehouse).id
         suggest_context = {
-            "order_id": po.id,
-            "domain": [],
             "warehouse_id": warehouse_id,
             "suggest_based_on": based_on,
             "suggest_percent": factor,
             "suggest_days": days,
         }
         po_id = self.env["purchase.order"].with_context(suggest_context).browse(po.id).ensure_one()
```

Comment:
Not sure to understand why this line ? You already have the purchase order (`po`, from the parameters), you can call `action_purchase_order_suggest` with given the wanted context